src/slim-select/index.ts

```diff
@@ -482,4 +486,23 @@ export default class SlimSelect {
       document.body.removeChild(slimContent)
     }
   }
+
+
+  private documentClick(e: Event): void {
```

Comment:
```suggestion
  private documentClick: (e: Event) => void = e => {
```

this would avoid the `bind` call, avoid `: any` on the new namespace object and still allow you to simply write

```
document.addEventListener('click', this.documentClick)
document.removeEventListener('click', this.documentClick)
```